src/pages/Signin/Signin.js

```diff
@@ -0,0 +1,97 @@
+import axios from 'axios'
+import React from 'react'
+import { useState } from 'react'
+import './Signin.css'
+import { Link, useNavigate } from 'react-router-dom';
+import { Toast } from '../../components/index';
+import { useData } from '../../Context/Context';
+import { useAuth } from '../../Context/AuthProvider';
```

Comment:
You can use the barrel export method to avoid multiple import lines.
Read it here https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno